app/views/index/normal.phtml

```diff
@@ -72,44 +73,160 @@ $today = @strtotime('today');
+												<?php
+												foreach ($remainingTags as $tag) {
+													?><li class="item"><a href="<?= _url('index', 'index', 'search', '#' . str_replace(' ', '+', htmlspecialchars_decode($tag, ENT_QUOTES))) ?>" title="<?= _t('gen.action.filter') ?>"><?= $tag ?></a></li><?php
+												} ?>
+											</ul>
+											<a class="dropdown-close" href="#close">❌</a>
+										</div>
+									</li>
+									<?php
+								} ?>
+								</ul><?php
+							} ?>
+						</div>
+						<?php
+					} ?>
+					<h1 class="title"><a target="_blank" rel="noreferrer" class="go_website" href="<?= $this->entry->link() ?>" title="<?= _t('conf.shortcut.see_on_website')?>"><?= $this->entry->title() ?><span class="extern-link-icon"><?= _i('link') ?></span></a></h1>
+					<?php if (FreshRSS_Context::$user_conf->show_author_date == 'h' || FreshRSS_Context::$user_conf->show_author_date == 'b') { ?>
+					<div class="subtitle">
+						<?php if (FreshRSS_Context::$user_conf->show_feed_name == 'a') { ?>
+							<div class="website"><a href="<?= _url('index', 'index', 'get', 'f_' . $this->feed->id()) ?>" title="<?= _t('gen.action.filter') ?>">
+								<?php if (FreshRSS_Context::$user_conf->show_favicons): ?>
+									<img class="favicon" src="<?= $this->feed->favicon() ?>" alt="✇" loading="lazy" /><?php
+								endif; ?><span><?= $this->feed->name() ?></span></a>
+							</div>
+							<?php
+						}
+
 						$authors = $this->entry->authors();
-						if (!empty($authors) && is_array($authors)):
-							$first = true;
-					?>
-					<div class="author"><?php
-							foreach ($authors as $author):
-								echo $first ? _t('gen.short.by_author') . ' ' : '· ';
-								$first = false;
-					?>
-	<em><a href="<?= Minz_Url::display(Minz_Request::modifiedCurrentRequest(
-			['search' => 'author:"' . htmlspecialchars_decode($author, ENT_QUOTES) . '"'])
-		) ?>"><?= $author ?></a></em>
-					<?php	endforeach; ?>
-					</div><?php endif; ?>
-				</div>
+						if (!empty($authors) && is_array($authors)) {
+						?>
+						<div class="author">
+							<?= _t('gen.short.by_author') ?>
+							<?php
+							foreach ($authors as $author) {
+								$href = Minz_Url::display(Minz_Request::modifiedCurrentRequest(['search' => 'author:"' . htmlspecialchars_decode($author, ENT_QUOTES) . '"']));
+							?>
+							<a href="<?= $href ?>" title="<?= _t('gen.action.filter') ?>"><?= $author ?></a>
+							<?php } ?>
+						</div>
+						<?php } ?>
+						<div class="date"><?= $this->entry->date() ?></div>
+					</div>
+					<?php } ?>
+				</header>
 				<div class="text"><?php
 					echo $lazyload && $hidePosts ? lazyimg($this->entry->content()) : $this->entry->content();
 				?></div>
-			</div><?php
-
-			$this->renderHelper('index/normal/entry_bottom');
-
-		?></div>
+				<footer>
+					<?php
+					if (FreshRSS_Context::$user_conf->show_author_date == 'h' || FreshRSS_Context::$user_conf->show_author_date == 'b') { ?>
```

Comment:
All those equalities against a normal string should use triple equal such as `=== 'f'`